airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/Application.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.container_orchestrator;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.airbyte.commons.logging.LoggingHelper;
+import io.airbyte.commons.logging.MdcScope;
+import io.airbyte.container_orchestrator.orchestrator.JobOrchestrator;
+import io.airbyte.workers.process.AsyncKubePodStatus;
+import io.micronaut.runtime.Micronaut;
+import jakarta.inject.Named;
+import jakarta.inject.Singleton;
+import java.lang.invoke.MethodHandles;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Entrypoint for the application responsible for launching containers and handling all message
+ * passing for replication, normalization, and dbt. Also, the current version relies on a heartbeat
+ * from a Temporal worker. This will also be removed in the future so this can run fully async.
+ * <p>
+ * This application retrieves most of its configuration from copied files from the calling Temporal
+ * worker.
+ * <p>
+ * This app uses default logging which is directly captured by the calling Temporal worker. In the
+ * future this will need to independently interact with cloud storage.
+ */
+@SuppressWarnings({"PMD.AvoidCatchingThrowable", "PMD.DoNotTerminateVM", "PMD.AvoidFieldNameMatchingTypeName"})
+@Singleton
+public class Application {
+
+  public static void main(final String[] args) {
+    // To mimic previous behavior, assume an exit code of 1 unless Application.run returns otherwise.
+    var exitCode = 1;
```

Comment:
Consider extracting this to a named constant, as it is declared here and again on line 83.